gdal/frmts/netcdf/frmt_netcdf.html

```diff
@@ -343,6 +343,17 @@ <h2>VSI Virtual File System API support</h2>
 Since GDAL 2.4, and with Linux kernel &gt;=4.3 and libnetcdf &gt;=4.5,
 read operations on /vsi file systems are supported.<p>
 
+<h2>NetCDF-4 groups support on reading (GDAL > 2.4.0)</h2>
+
+The driver has undergone significant changes after GDAL 2.4.0 to support NetCDF-4 groups on reading, see NEWS file.
```

Comment:
do we really need the reference to NEWS file ? I think the below description will be far more complete than what I will be able to write myself

Reply:
"GDAL >= 2.5.0"  There will probably 2.4.x maintenance releases, but this work is not appropriate for them.